server/sonar-web/src/main/js/api/permissions.js

```diff
@@ -178,3 +202,28 @@ export function getGlobalPermissionsGroups (query = '', permission = null) {
   }
   return getJSON(url, data).then(r => r.groups);
 }
+
+export function getPermissionTemplateUsers (templateId, query = '', permission = null) {
+  const url = '/api/permissions/template_users';
+  const data = { templateId, ps: PAGE_SIZE };
+  if (query) {
+    data.q = query;
+  }
+  if (permission) {
+    data.permission = permission;
+  }
+  return getJSON(url, data).then(r => r.users);
+}
+
+export function getPermissionTemplateGroups (templateId, query = '', permission = null) {
+  // FIXME change to /template_groups
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1134)